localfs/store.go

```diff
@@ -278,11 +279,32 @@ func (l *LocalStore) Get(ctx context.Context, o string) (cloudstorage.Object, er
 // Delete the object from underlying store.
 func (l *LocalStore) Delete(ctx context.Context, obj string) error {
 	fo := path.Join(l.storepath, obj)
-	os.Remove(fo)
+	if err := os.Remove(fo); err != nil {
+		return err
```

Comment:
[NIT] Should we start wrapping these errors?

```suggestion
		return fmt.Errorf("removing obj: %w", err)
```

Same below.